ang/civicase/CustomScrollbar.js

```diff
@@ -0,0 +1,62 @@
+(function (angular, $, _) {
+  var module = angular.module('civicase');
+
+  module.directive('civicaseCustomScrollbar', function ($rootScope, $parse, $timeout) {
+    return {
+      restrict: 'A',
+      link: civicaseCustomScrollbarLink
+    };
+
+    /**
+     * Link function for civicaseCustomScrollbar
+     *
+     * @param {Object} $scope
+     * @param {jQuery} element
+     * @param {Object} attrs
+     */
+    function civicaseCustomScrollbarLink ($scope, element, attrs) {
+      var options = {
+        autoHide: false
+      };
+
+      if (attrs.civicaseCustomScrollbar) {
+        options = $parse(attrs.civicaseCustomScrollbar)();
```

Comment:
what if `attrs.civicaseCustomScrollbar` is 
```js
{
  someProp: 'someValue'
}
```
will the default `autoHide` property be removed from the original `options`?

Reply:
Yes.

Reply:
so the implementation is incorrect, why would you delete the default `autoHide` value if you are not passing a new value to the directive? It looks like you should either use `_.defaults()` or `_.assign()`